index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
```

Comment:
It would be nice to have a header section to delimit the title correctly.

Reply:
- [ ]  This is an amazing recommendation @melaniesigrid. But don't you feel like it would be better if you explained why putting the title portion in a header tag is more semantic? Accessibility and information flow in website are just two of the things semantic tags help the website with and explaining it to the reviewer will make him understand why you're requesting these changes. Always remember to add a "Why" when you request a change
- [ ] Remember to select the exact list of lines you're commenting on instead of a single line like  [this](https://github.com/melaniesigrid/past_project_review/pull/2/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R11-R14). If you need help on how to do it you can use this [resource](https://github.blog/changelog/2019-10-01-multi-line-references-and-comments-beta/) .